content/en/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -31,7 +31,14 @@ <h3>Scaling an application</h3>
             <p>If you haven't worked through the earlier sections, start from <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">Using minikube to create a cluster</a>.</p>
 
             <p><em>Scaling</em> is accomplished by changing the number of replicas in a Deployment</p>
-            <p> <b> NOTE </b> If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section </a>, you may need to start from <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/">creating a cluster</a> as the services may have been deleted </p>
+            <p><b>NOTE</b> If you are trying this after <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/">the previous section </a>, then you
+            may have deleted the Sservice you created, or have created a Service of <tt>type: NodePort</tt>.
```

Comment:
What is `Sservice` here i think you wants to write `service`.

Reply:
@niranjandarshann Fixed the type